ReactWindows/ReactNative.Shared/Bridge/ReactBridge.cs

```diff
@@ -108,11 +110,35 @@ public void Dispose()
 
         private void ProcessResponse(JToken response)
         {
+            ProcessResponse(response, false);
+        }
+
+        private void ProcessResponse(JToken response, bool isEndOfBatch)
+        {
+            bool localBatchHadNativeModuleCalls;
+
             if (response == null || response.Type == JTokenType.Null || response.Type == JTokenType.Undefined)
             {
+                if (isEndOfBatch)
+                {
+                    localBatchHadNativeModuleCalls = _batchHadNativeModuleCalls;
+
+                    _nativeModulesQueueThread.Dispatch(() =>
+                    {
+                        // the bridge could still be processing native calls when the idle signaler fires
+                        if (localBatchHadNativeModuleCalls)
```

Comment:
I don't understand this comment, can you expand?

Reply:
Maybe it's only related to the bridge idle feature...?

---
In reply to: [181254167](https://github.com/Microsoft/react-native-windows/pull/1769#discussion_r181254167) [](ancestors = 181254167)